modules/proc_configuring-annotation-based-tracking-in-multiple-argo-cd-instances.adoc

```diff
@@ -0,0 +1,146 @@
+<2> Specifies the namespace used for the second Argo CD instance.
+<3> Specifies the name of the `installationID` object for the second Argo CD instance.
+
+. Configure and label target namespaces to associate namespaces with their Argo CD instances.
+.. Navigate to *Administration* -> *Namespaces*.
+.. Create namespaces for application deployments, `app-ns-1` and `app-ns-2`.
+.. Associate each namespace with their respective Argo CD instance:
+... Associate the `app-ns-1` namespace with the `argocd-test-demo-1` Argo CD instance by running the following command: 
++
+.Example command
+[source,terminal]
+----
+$ oc label namespace app-ns-1 argocd.argoproj.io/managed-by=argocd-test-demo-1
+----
+... Associate the `app-ns-2` namespace with the `argocd-test-demo-2` Argo CD instance by running the following command: 
++
+.Example command
+[source,terminal]
+----
+$ oc label namespace app-ns-2 argocd.argoproj.io/managed-by=argocd-test-demo-2
+----
++
+. Create two applications in Argo CD.
+.. In the {OCP} web console, go to *Operators* -> *Installed Operators* -> *OpenShift GitOps Operator*.
+.. Select *Argo CD* and navigate to the *Applications* tab.
+.. Click *Create Application*.
+.. Enter the following YAML snippet to create two applications in Argo CD.
++
+.Example first application using Argo CD
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sprint-petclinic # <1>
+  namespace: argocd-test-demo-1 # <2>
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/redhat-developer/gitops-operator
+    path: test/examples/nginx
+    targetRevision: HEAD
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: app-ns-1
+  syncPolicy:
+    automated: {}
+----
+<1> Specifies the name of the first application.
+<2> Specifies the namespace used for the first application.
++
+.Example second application using Argo CD
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sprint-petclinic # <1>
+  namespace: argocd-test-demo-2 # <2>
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/redhat-developer/gitops-operator
+    path: test/examples/nginx
+    targetRevision: HEAD
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: app-ns-2
+  syncPolicy:
+    automated: {}
+----
+<1> Specifies the name of the second application that is created with the same name as the first application.
+<2> Specifies the namespace used for the second application.
+
+.Verification
+. Navigate to *Workloads* -> *Pods* in the {OCP} web console.
+. Ensure that the pods for Argo CD instances `argocd-instance-demo-1` and `argocd-instance-demo-2` are running.
+. Check the application synchronization status in the *Argo CD Applications* YAML tab.
+. Navigate to the `argocd-cm` config map in `argocd-test-demo-1` and `argocd-test-demo-2` namespaces and verify that the `installationID`object is configured successfully.
```

Comment:
Tiny typo here impacting the HTML presentation:
```suggestion
. Navigate to the `argocd-cm` config map in `argocd-test-demo-1` and `argocd-test-demo-2` namespaces and verify that the `installationID` object is configured successfully.
```

Reply:
Fixed.